src/datepicker/datepicker-month-view.spec.ts

```diff
@@ -107,6 +107,22 @@ describe('ngbDatepickerMonthView', () => {
     expect(fixture.componentInstance.onClick).not.toHaveBeenCalledWith();
   });
 
+  it('should not send date selection events if disabled', () => {
+    const fixture = createTestComponent(`
+        <template #tpl let-date="date">{{ date.day }}</template>
+        <tbody ngbDatepickerMonthView [month]="month" [dayTemplate]="tpl" [disabled]="true" (select)="onClick($event)"></tbody>
+      `);
+
+    fixture.detectChanges();
+
+    spyOn(fixture.componentInstance, 'onClick');
+
+    const dates = getDates(fixture.nativeElement);
+    dates[0].click();
+
+    expect(fixture.componentInstance.onClick).not.toHaveBeenCalledWith();
```

Comment:
better be `not.toHaveBeenCalled()` but I guess it's my mistake from the test case above (line 107) :)